cgoutils/memory.go

```diff
@@ -35,6 +35,10 @@ func IsPooledMemory() bool {
 	return (GetFlags() & C.POOLED_MEMORY_FLAG) != 0
 }
 
+func SupportHashReduction() bool {
```

Comment:
feels like `SupportHashReduction` is not part of `memory.go`..

Reply:
well, there is no better place to put those flags ..